service-api/module/Application/test/ApplicationTest/Services/Experian/FraudApi/DTO/ExperianCrosscoreFraudResponseDTOTest.php

```diff
@@ -0,0 +1,255 @@
+<?php
+
+declare(strict_types=1);
+
+namespace ApplicationTest\ApplicationTest\Services\Experian\FraudApi\DTO;
+
+use Application\Experian\Crosscore\FraudApi\DTO\ResponseDTO;
+use PHPUnit\Framework\TestCase;
+
+class ExperianCrosscoreFraudResponseDTOTest extends TestCase
+{
+    private ResponseDTO $experianCrosscoreFraudResponseDTO;
+
+    private array $data;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        $this->data = [
```

Comment:
You can minimise this to just the bits that we actually use, since everything else is ignored.

There's the same in a few other test files. At a minimum, I think the `clientResponsePayload` and `originalRequestData` bits can be removed everywhere.